ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productcarousel/v1/productcarousel/productcarousel.html

```diff
@@ -0,0 +1,25 @@
+
+<sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html" 
+	 data-sly-call="${clientlib.all @ categories='aem.venia.productcarousel'}"/>
+
+<sly data-sly-test="${!properties.product}">
+    <div class="blank-placeholder product-carousel-title" data-sly-text="${component.title}"></div>
+</sly>
+
+<sly data-sly-test="${properties.product}">
+    <div class="product-carousel carousel-root">
+        <div class ="product-carousel-parent" data-sly-use.productList="${'com.adobe.cq.commerce.core.components.models.productcarousel.ProductCarousel' @productSkuList=properties.product}">
+            <div class="cards-container"  data-sly-list.product=${productList.products}>
+		    <div class="card product-card">
+                    	<a href="${product.URL}">
+                            <img style="width:240px;height:240px;"  src="${product.imageURL}"  data-gallery-role="currentimage"/>
```

Comment:
This belongs in CSS. 
If you need the value in your javascript then define a constant there.

Reply:
style moved to css class